scripts/ci/testing/ci_run_single_airflow_test_in_docker.sh

```diff
@@ -123,8 +124,10 @@ function run_airflow_testing_in_docker() {
       "${DOCKER_COMPOSE_LOCAL[@]}" \
       --project-name "airflow-${TEST_TYPE}-${BACKEND}" \
          run airflow "${@}"
+    docker ps
```

Comment:
Whoops! Putting this here meant that the exit_code of each test type was always 0.

Reply:
I will file a fix (remove that) if you are working on it yet @ashb 
@potiuk are you  ok with this removal?

Reply:
Created this for the same: https://github.com/apache/airflow/pull/19682

Reply:
Oh yeah. That was a sneaky attempt to make our tests green again :scream: . Yeah. My bad.